lib/minify.js

```diff
@@ -0,0 +1,125 @@
+exports.readFile = exports.simple_glob = exports.readNameCache =
+    exports.writeNameCache = function() {
+        throw Error("UglifyJS has no access to the file system");
+    }
+
+var minify = function(files, options) {
+    options = defaults(options, {
+        spidermonkey     : false,
+        outSourceMap     : null,
+        sourceRoot       : null,
+        inSourceMap      : null,
+        sourceMapUrl     : null,
+        sourceMapInline  : false,
+        fromString       : false,
+        warnings         : false,
+        mangle           : {},
+        mangleProperties : false,
+        nameCache        : null,
+        output           : null,
+        compress         : {},
+        parse            : {}
+    });
+    base54.reset();
+
+    // 1. parse
+    var toplevel = null,
+        sourcesContent = {};
+
+    if (options.spidermonkey) {
+        toplevel = AST_Node.from_mozilla_ast(files);
+    } else {
+        function addFile(file, fileUrl) {
```

Comment:
Actually, this is the same code. And didn't fix (although I could) because I spend time doing other things apparently.